sippy-ng/README.md

```diff
@@ -19,6 +19,9 @@ Formatting requirements are enforce on the order of imports (alphabetically) and
 sippy/sippy-ng: $ npx prettier -w src/
 ```
 
+For VScode, users install the [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) plugin; when the you see errors tagged with `prettier/prettier`, use the CMD/CTRL + Shift + P keys
+followed by "Format Document" to quickly fix them.
```

Comment:
Were you not able to get formatting on save to work? https://www.robinwieruch.de/how-to-use-prettier-vscode/

Reply:
There's an option to configure on save formatting but I'm hesitant to turn it on just yet until I see what it will format for my other non js files.